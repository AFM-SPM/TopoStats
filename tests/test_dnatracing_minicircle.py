"""Tests the dnatracing module"""
from pathlib import Path
import numpy as np
import pytest

import pandas as pd


def test_dnatracing_minicircle(minicircle_dnatracing, minicircle_dnastats) -> None:
    pd.testing.assert_frame_equal(minicircle_dnatracing, minicircle_dnastats)
