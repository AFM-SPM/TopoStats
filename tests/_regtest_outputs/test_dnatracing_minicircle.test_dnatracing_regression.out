                 Contour Lengths  Circular  End to End Distance Image Name Basename
Molecule Number                                                                    
0                     212.564742     False            35.909630        tmp        .
1                      45.518573      True             0.000000        tmp        .
2                     136.474012     False            27.438294        tmp        .
3                      95.602886      True             0.000000        tmp        .
4                      97.595102     False            25.901022        tmp        .
5                      98.166730      True             0.000000        tmp        .
6                      88.271594     False            22.551653        tmp        .
7                      98.828601      True             0.000000        tmp        .
8                     113.327558     False            23.432616        tmp        .
9                     103.446601      True             0.000000        tmp        .
10                     76.360523      True             0.000000        tmp        .
11                     82.888759      True             0.000000        tmp        .
12                    105.799458     False            27.844417        tmp        .
13                    101.795150      True             0.000000        tmp        .
14                     99.466268      True             0.000000        tmp        .
15                     64.998424      True             0.000000        tmp        .
16                     99.739014     False            27.703832        tmp        .
17                    106.942811      True             0.000000        tmp        .
18                     71.480238      True             0.000000        tmp        .
19                    101.777703      True             0.000000        tmp        .
20                     98.084923      True             0.000000        tmp        .
21                     99.992938     False            20.451960        tmp        .
